storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw_test.go

```diff
@@ -693,6 +693,146 @@ func sortTimeSeries(series []prompb.TimeSeries) []prompb.TimeSeries {
 	return series
 }
 
+func TestTranslatorMetricFromOtelMetric(t *testing.T) {
```

Comment:
Should some of the tests apply different temporality, since that is what the previous code looked at?